lib/ansible/modules/crypto/get_certificate.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: get_certificate
+author: "John Westcott IV"
+version_added: "2.7"
+short_description: Get a certificate from a host:port
+description:
+    - Makes an SSL connection and returns information about the presented certificate
+options:
+    host:
+      description:
+        - The host to get the cert for (IP is fine)
+      required: True
+    ca_certs:
+      description:
+        - A file containing a list of root certificates; if present, the cert will be validated against these root certs.
```

Comment:
Which format is this file? PEM?

Also, this parameter only accepts a single file, maybe writing `ca_cert` might be better?

Reply:
Or `ca_cert file`?

Reply:
Yes. Per the docs: https://docs.python.org/2/library/ssl.html#ssl.wrap_socket. I've added notation to indicate its a PEM file.

Are you thinking that we should take the contents of the PEM itself and then write out the CA file to be passed into the function? I think I'd rather have a user have another Ansible step to write the file that we can then use in a get_certificate step.

Reply:
I think it should be enough to document that it is a PEM file, not just "a" file.

Reply:
Fair enough, I will take certs out for now.